Themis/Views/Assessment/CorrectionSidebar/FeedbackListView.swift

```diff
@@ -113,7 +113,7 @@ struct FeedbackListView: View {
 }
 
  struct FeedbackListView_Previews: PreviewProvider {
-    static let assessment = AssessmentViewModel(readOnly: false)
+     static let assessment = AssessmentViewModel(readOnly: false)
```

Comment:
```suggestion
    static let assessment = AssessmentViewModel(readOnly: false)
```

What does the fifth space do here? :D